lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -83,8 +84,14 @@
                                 model: userAccountModel,
                                 title: gettext('Full Name'),
                                 valueAttribute: 'name',
-                                helpMessage: gettext(
-                                    'The name that is used for ID verification and appears on your certificates. Other learners never see your full name. Make sure to enter your name exactly as it appears on your government-issued photo ID, including any non-Roman characters.'  // eslint-disable-line max-len
+                                helpMessage: HtmlUtils.interpolateHtml(
+                                    gettext(
```

Comment:
I believe the whole `gettext` call has to be on a single line for the scraper to work.

Reply:
Done.  I also squashed.